### ISSUE TITLE
Cannot load additional assets

### ISSUE THREAD

**Issue by jae1911** [Yellow-Dog-Man/Resonite-Issues#780]
### Describe the bug?

Assets won't load and logs are spammed by error messages about the localdb.
Cached assets load fine.

### To Reproduce

Update game to `2023.11.24.1379` and start the game.

### Expected behavior

Assets should load.

### Screenshots

![Resonite_E9YVdtIXi8](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76598503/34d84a6f-374a-4385-aac7-b3b224451ee0)

### Resonite Version Number

2023.11.24.1379

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Index

### Log Files

[J4 - 2023.11.24.1379 - 2023-11-25 01_33_05.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13462557/J4.-.2023.11.24.1379.-.2023-11-25.01_33_05.log)

### Reporters

j4 | j4.lc (Discord)

**Comment by shadowpanther** [comment]
The log is full of this exception repeating for every GatherJob:
```
00:40:09.685 ( 60 FPS)	Exception running GatherJob: GatherJob resdb:///f774c0771d6773ca99c4a5305333bb9cd66318920b1e413c8bdadfb776ee23da?version=3&compression=BC1_Crunched&quality=100&width=128&height=64&mips=True&filtering=Box&colorPreprocess=None&alphaPreprocess=None, State: Finished, StatusCode: OK, Progress: 100.00, Started: 24/11/2023 23:40:09
System.NullReferenceException: Object reference not set to an instance of an object
  at FrooxEngine.Store.LocalDB+<StoreCacheRecordAsync>d__87.MoveNext () [0x00040] in <bf7d8063373d47fcaa14cccfb3111b65>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter[TResult].GetResult () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at FrooxEngine.EngineGatherJob+<FinishGather>d__16.MoveNext () [0x00089] in <70a16d0333534b3fbb9dacef1fc58637>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter[TResult].GetResult () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at SkyFrost.Base.GatherJob+<Download>d__83.MoveNext () [0x00137] in <3b9a09257a664165a86f90f223ecbf9d>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.ConfiguredTaskAwaitable+ConfiguredTaskAwaiter.GetResult () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at SkyFrost.Base.AssetGatherer`1+<ProcessJob>d__7[G].MoveNext () [0x0012d] in <3b9a09257a664165a86f90f223ecbf9d>:0 

  at System.Environment.get_StackTrace () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at Elements.Core.UniLog.Error (System.String message, System.Boolean stackTrace) [0x00000] in <fde91e37a09f4b15ac96136011f003b7>:0 
  at SkyFrost.Base.AssetGatherer`1+<ProcessJob>d__7[G].MoveNext () [0x00000] in <3b9a09257a664165a86f90f223ecbf9d>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.InvokeMoveNext (System.Object stateMachine) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction (System.Action action, System.Boolean allowInlining, System.Threading.Tasks.Task& currentTask) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishContinuations () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishStageThree () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishStageTwo () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.Finish (System.Boolean bUserDelegateExecuted) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task`1[TResult].TrySetException (System.Object exceptionObject) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[TResult].SetException (System.Exception exception) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.SetException (System.Exception exception) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at SkyFrost.Base.GatherJob+<Download>d__83.MoveNext () [0x00000] in <3b9a09257a664165a86f90f223ecbf9d>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.InvokeMoveNext (System.Object stateMachine) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction (System.Action action, System.Boolean allowInlining, System.Threading.Tasks.Task& currentTask) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishContinuations () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishStageThree () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task`1[TResult].TrySetResult (TResult result) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[TResult].SetResult (TResult result) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[TResult].SetResult (System.Threading.Tasks.Task`1[TResult] completedTask) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.SetResult () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at SkyFrost.Base.GatherJob+<RunDownload>d__89.MoveNext () [0x00000] in <3b9a09257a664165a86f90f223ecbf9d>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.InvokeMoveNext (System.Object stateMachine) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction (System.Action action, System.Boolean allowInlining, System.Threading.Tasks.Task& currentTask) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishContinuations () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishStageThree () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task`1[TResult].TrySetResult (TResult result) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.TaskFactory`1+FromAsyncTrimPromise`1[TResult,TInstance].Complete (TInstance thisRef, System.Func`3[T1,T2,TResult] endMethod, System.IAsyncResult asyncResult, System.Boolean requiresSynchronization) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.TaskFactory`1+FromAsyncTrimPromise`1[TResult,TInstance].CompleteFromAsyncResult (System.IAsyncResult asyncResult) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.IO.Stream+ReadWriteTask.InvokeAsyncCallback (System.Object completedTask) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.IO.Stream+ReadWriteTask.System.Threading.Tasks.ITaskCompletionAction.Invoke (System.Threading.Tasks.Task completingTask) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishContinuations () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishStageThree () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.FinishStageTwo () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.Finish (System.Boolean bUserDelegateExecuted) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.ExecuteWithThreadLocal (System.Threading.Tasks.Task& currentTaskSlot) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.ExecuteEntry (System.Boolean bPreventDoubleExecution) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading.ThreadPoolWorkQueue.Dispatch () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
```
[VIXEN - 2023.11.24.1379 - 2023-11-25 00_38_58.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13462574/VIXEN.-.2023.11.24.1379.-.2023-11-25.00_38_58.log)

Tried database repair - no noticeable effect.

**Comment by Frooxius** [comment]
Looking into this, will try to post hotfix ASAP.

**Comment by AegisTheWolf** [comment]
yup, so far people have sync errors, are unable to load certain items, and some are set to invisible for some reason.

**Comment by Zyro1331** [comment]
I'm having similar issues on a separate machine, most content is cached but things like the Cloud Home, World Thumbnails, and other things in the UI are not loading correctly. 

[STEAMMACHINE - 2023.11.24.1379 - 2023-11-24 16_07_02.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13462616/STEAMMACHINE.-.2023.11.24.1379.-.2023-11-24.16_07_02.log)
Here's the logs from my machine.

**Comment by ProbablePrime** [comment]
Froox has provided an update here: https://discord.com/channels/1040316820650991766/1154514007479287942/1177763624278827008

That we should have a fix within 5 minutes.

**Comment by Frooxius** [comment]
Fix has been pushed: 2023.11.25.12

Thank you for the report!
